Source/cmake/OptionsGTK.cmake

```diff
@@ -61,6 +61,7 @@ WEBKIT_OPTION_DEFINE(USE_OPENGL_OR_ES "Whether to use OpenGL or ES." PUBLIC ON)
 WEBKIT_OPTION_DEFINE(USE_OPENJPEG "Whether to enable support for JPEG2000 images." PUBLIC ON)
 WEBKIT_OPTION_DEFINE(USE_SOUP2 "Whether to enable usage of Soup 2 instead of Soup 3." PUBLIC OFF)
 WEBKIT_OPTION_DEFINE(USE_WOFF2 "Whether to enable support for WOFF2 Web Fonts." PUBLIC ON)
+WEBKIT_OPTION_DEFINE(USE_LIBBACKTRACE "Whether to enable usage of libbacktrace." PUBLIC OFF)
```

Comment:
I alphabetize it. I can not turn it ON for now because it will break the bots. `libbacktrace` is a separate library and it's not preinstalled yet. I'm going to Flatpak SDK and ghbuild to include it in follow-up PRs. When it's done we can turn the flat on by default.